api/database/seeders/UserTestSeeder.php

```diff
@@ -98,5 +102,40 @@ public function run()
                 'email' => 'noroles@test.com',
                 'sub' => 'noroles@test.com',
             ]);
+
+        User::factory()
+            ->asApplicant()
+            ->withSkillsAndExperiences()
```

Comment:
This is missing asProcessOperator. Also, I don't think this (or the community role users) need withSkillsAndExperiences.

Reply:
The process operator role is added in the PoolTestSeeder as the pool (and it's related team) don't exist yet. If you'd prefer I could probably update things to use a team defined in the TeamSeeder instead.

Removed withSkillsAndExperiences from the new users in [8739d66](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10621/commits/8739d66b94b5afb5c37fe571d9e5e8167a78db6a).

Reply:
Hmmm. Yes, I think you'd prefer to add it here. That would make it consistent with the Community-based roles and the PoolOperator role.

Reply:
Changed seeding to attach the process operator role here in [533700e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10621/commits/533700e6cc4a3f20ab0b959ead9279da18604343).